superset-frontend/src/views/CRUD/welcome/DashboardTable.tsx

```diff
@@ -69,6 +73,13 @@ function DashboardTable({
   const [editModal, setEditModal] = useState<Dashboard>();
   const [dashboardFilter, setDashboardFilter] = useState('Mine');
 
+  useEffect(() => {
+    const filter = getFromLocalStorage('dashboard', null);
```

Comment:
`homepage_dashboard_filter`?

Reply:
Also, these keys could be extracted into constants

Reply:
👍 on keeping these in a constants file or using an enum. It’ll make it much easier to add a new key without risk of collision with an existing one